packages/medusa/src/api/admin/order-changes/[id]/route.ts

```diff
@@ -0,0 +1,33 @@
+import { setCarryOverPromotionFlagForOrderChangeWorkflow } from "@medusajs/core-flows"
+import { HttpTypes } from "@medusajs/framework/types"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+  refetchEntity,
+} from "@medusajs/framework/http"
+import { AdminPostOrderChangesReqSchemaType } from "../validators"
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminPostOrderChangesReqSchemaType>,
+  res: MedusaResponse<HttpTypes.AdminOrderChangeResponse>
+) => {
+  const { id } = req.params
+  const { carry_over_promotions } = req.validatedBody
+
+  const workflow = setCarryOverPromotionFlagForOrderChangeWorkflow(req.scope)
```

Comment:
nit: feels a bit off to use a very workflow in a generic update API route, should we create (or rename) an `updateOrderChangesWorkflow` instead?